pkg/util/sys/device.go

```diff
@@ -143,12 +149,12 @@ func GetDevicePropertiesFromPath(devicePath string, executor exec.Executor) (map
 // get the file systems availab
 func GetDeviceFilesystems(device string, executor exec.Executor) (string, error) {
 	cmd := fmt.Sprintf("get filesystem type for %s", device)
-	output, err := executor.ExecuteCommandWithOutput(false, cmd, "df", "--output=source,fstype")
+	output, err := executor.ExecuteCommandWithOutput(false, cmd, "wipefs", "-p", "-n", fmt.Sprintf("/dev/%s", device))
```

Comment:
wipefs will detect the fs whether or not it is mounted?

Reply:
yes, it reads off disk